grid_map_sdf/include/grid_map_sdf/distance_transform/pnmfile.h

```diff
@@ -18,20 +18,23 @@ Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307 USA
 
 /* basic image I/O */
 
+#ifndef GRID_MAP_SDF__DISTANCE_TRANSFORM__PNMFILE_H_
+#define GRID_MAP_SDF__DISTANCE_TRANSFORM__PNMFILE_H_
+
 #pragma once
 
 #include <cstdlib>
 #include <climits>
 #include <cstring>
 #include <fstream>
 
-#include "image.h"
-#include "misc.h"
+#include "grid_map_sdf/distance_transform/image.h"
+#include "grid_map_sdf/distance_transform/misc.h"
 
 namespace distance_transform
 {
 
-  const short BUF_SIZE = 256;
+  const int16 BUF_SIZE = 256;
```

Comment:
I can see it compiling fine, but I am not sure if other compiler will be happy without "_t" in `int16`.

Reply:
will do, sorry that one was missed in my second run through